spec/directives/extend/across_modules.hrx

```diff
@@ -0,0 +1,74 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+<==============================================================================>
+<===> extending_an_extender_multiple_times/extendee.scss
+.extendee {a: 1}
+
+<===> extending_an_extender_multiple_times/extender.scss
+@use "extendee";
+.extender {@extend .extendee}
+
+<===> extending_an_extender_multiple_times/extender_of_extender1.scss
+@use "extender";
+.extender_of_extender1 {
+  @extend .extender;
+  b: 1;
+}
+
+<===> extending_an_extender_multiple_times/extender_of_extender2.scss
+@use "extender";
+.extender_of_extender2 {
+  @extend .extender;
+  b: 2;
+}
+
+<===> extending_an_extender_multiple_times/input.scss
+@use "extender_of_extender1";
+@use "extender_of_extender2";
```

Comment:
In other diamond dependency tests we call these `left`, `right`, and `downstream`... might be good to use the same convention here.